.github/workflows/run_release.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
         draft: true
         prerelease: false
         name: ${{ inputs.releaseName }}
-        tag_name: ${{ inputs.version }}
+        tag_name: v${{ inputs.version }}
```

Comment:
Can you please update the definition of this parameter on line 9 to provide an example (so people know they're supposed to leave out the v)?